frontend-v2/src/styles/globals.css

```diff
@@ -23,6 +23,20 @@ body {
   -moz-background-size: cover;
   -o-background-size: cover;
   background-size: cover;
+  scrollbar-gutter: stable;
+}
+
+html {
+  /* Reduce jitter when Shadcn <Select> component displays a modal that hides the scrollbar.
+     https://github.com/radix-ui/primitives/issues/1496#issuecomment-1962839229 */
+  scrollbar-gutter: stable;
+}
+
+/* Reduce jitter when Shadcn <Select> component displays a modal that hides the scrollbar.
+   https://github.com/radix-ui/primitives/issues/1496#issuecomment-1962839229 */
+html body[data-scroll-locked] {
```

Comment:
does the entire page jump up and down for you when the <Select> opens and closes? not sure if that is what you were trying to fix w/ this or it's unrelated.

Reply:
yes, and this fixes it a little bit but only partway. without this, the scrollbar disappears and the whole page gets a little bit wider.
clarified in the comments.

Reply:
gotcha, my computer doesn't show scrollbars unless i'm scrolling.